libs/blocks/notification/notification.js

```diff
@@ -135,6 +135,9 @@ async function decorateLockup(lockupArea, el) {
 async function decorateForegroundText(el, container) {
   const text = container?.querySelector('h1, h2, h3, h4, h5, h6, p')?.closest('div');
   text?.classList.add('text');
+  if (el.classList.contains('countdown-timer') && !el.matches('.pill, .ribbon')) {
```

Comment:
Let's make this more consistent and use one approach for both checks, either `classList.contains` or `.matches`

Reply:
Should this logic go down in the decorateLayout() function where these conditional checks are also already being done?

Reply:
I will have to parse the text div again